From b2aa209dfc5e59d6329b55b9764782334b63dbe8 Mon Sep 17 00:00:00 2001
From: Raphael Freudiger <raphael.freudiger@siemens.com>
Date: Wed, 11 Feb 2015 09:11:47 +0100
Subject: [PATCH] do not override compiler and do not strip

The default makefile sets the compiler to g++ or gcc. This leads to a wrong architecture when cross-compiling.
Remove the hardcoded compiler and just append the flags to CXX and CC.

Upstream-Status: Pending
Signed-off-by: Raphael Freudiger <raphael.freudiger@siemens.com>
Reviewed-By: Pascal Bach <pascal.bach@siemens.com>

---
 makefile.machine | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/makefile.machine b/makefile.machine
index 9e34c34..e9244d9 100644
--- a/makefile.machine
+++ b/makefile.machine
@@ -2,7 +2,7 @@
 # makefile for Linux (x86, PPC, alpha ...)
 #
 
-OPTFLAGS=-O -s
+OPTFLAGS=-O
 
 ALLFLAGS=${OPTFLAGS} -pipe \
     	-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE \
@@ -10,8 +10,6 @@ ALLFLAGS=${OPTFLAGS} -pipe \
 	-D_7ZIP_LARGE_PAGES \
 	$(LOCAL_FLAGS)
 
-CXX=g++
-CC=gcc
 CC_SHARED=-fPIC
 LINK_SHARED=-fPIC -shared
 
